packages/docs/content/components/badge.mdx

```diff
@@ -89,3 +89,20 @@ The following Sass variables can be overridden to customize Badge components:
 ### Future research
 
 - Further usability testing should be done to ensure badges aren't being confused with buttons. Further iteration can be done with sizing and colors if there is confusion.
+
+## Component maturity
+
+<MaturityChecklist
+  a11yStandards={true}
+  color={true}
+  forcedColors={true}
+  screenReaders={true}
+  keyboardNavigable={null}
```

Comment:
I've been setting keyboard-navigable to N/A for all components that are non-interactive. Can anyone think of any counter-examples? Maybe just pay attention to this field as you walk through these components.